pkg/reconciler/v1alpha1/namespace/namespace.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Knative Authors
+Copyright 2019 The Knative Authors
```

Comment:
2019 might break the build. It happened to me sometime back because there were some checks that are run against the boilerplate. I guess we will find out once you merge

Reply:
That would be unfortunate...I guess I'll find out when I try to merge upstream!